modules/API/external/external_api.js

```diff
@@ -366,6 +366,29 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     *
+     * @param {string} participantId - Participant id or "local" for local video.
+     *
+     * @returns {HTMLElement|undefined} - The requested video.
+     */
+    _getParticipantVideo(participantId) {
+        const iframe = this.getIFrame();
+
+        if (!iframe
+                || !iframe.contentWindow
+                || !iframe.contentWindow.document) {
+            return;
+        }
+
+        if (participantId == "local") {
```

Comment:
1. Another eslint error because of `==`
2. Isn't it going to be better to use `typeof participantId === 'undefined'` instead of requiring the consumer to pass a string. WDYT? If you agree you should change the JSDoc as well.
3. Let's also add to the check the use case when `participantId === this._myUserID`. This way you can call the method also with the id of the local user and still retrieve the correct video element.

Reply:
Thanks I didn't do npm install. Will check for eslint errors on my side. And yes I thought about calling with _myUserID. Nice, will implement that too.

Reply:
Yep. After you run `npm install` the eslint and flow validation will run automatically on every `git commit`.